ci.sh

```diff
@@ -112,9 +112,6 @@ else
     INSTALLDIR=$(python -c "import os, trio; print(os.path.dirname(trio.__file__))")
     cp ../pyproject.toml $INSTALLDIR
 
-    # TODO: remove this once we have a py.typed file
-    touch "$INSTALLDIR/py.typed"
-
```

Comment:
If being a stickler I'd remove all the instances of generating/removing `py.typed` in a dedicated PR.